src/material/menu/menu-trigger.ts

```diff
@@ -407,8 +407,16 @@ export abstract class _MatMenuTriggerBase implements AfterContentInit, OnDestroy
     menu.parentMenu = this.triggersSubmenu() ? this._parentMaterialMenu : undefined;
     menu.direction = this.dir;
     this._setMenuElevation(menu);
-    menu.focusFirstItem(this._openedBy || 'program');
-    this._setIsMenuOpen(true);
+
+    if (this._ngZone) {
```

Comment:
I'm guessing that we need this because of the slight delay. If that's the case, an alternative could be to do it inside `MatMenu._onAnimationDone`. The advantage of the alternate approach is that it's easier to flush in tests.

Reply:
Exactly, this looks like a timing problem. Adding a bit of delay from when the menu items get added to the DOM to when we call `.focus()` seems to work better with iOS Voice Over. I'll give `MatMenu._onAnimationDone` a shot, ty.